zerver/lib/actions.py

```diff
@@ -638,7 +638,7 @@ def do_start_email_change_process(user_profile, new_email):
     activation_url = EmailChangeConfirmation.objects.get_link_for_object(obj, host=user_profile.realm.host)
     context = {'realm': user_profile.realm, 'old_email': old_email, 'new_email': new_email,
                'activate_url': activation_url}
-    send_email('zerver/emails/confirm_new_email', new_email, from_email=settings.DEFAULT_FROM_EMAIL,
```

Comment:
Since this email has a confirmation link, do we want to make it come from the noreply email address so people don't accidentally send secret links to the administrators?